packages/tx-construction/src/tx-builder/OutputBuilder.ts

```diff
@@ -1,9 +1,10 @@
-import { Cardano } from '@cardano-sdk/core';
```

Comment:
Please add tests for the functional changes in `tx-construction` package

Reply:
I fixed the tests in tx-constructions, according to the changes made in this PR. However, I don't think the context output is tested. I might be wrong.

Reply:
The new functionality that we need to add tests for is:
- building outputs with a `handle`
- returning used handles in `ctx` together with signed transaction